test/view_component/integration_test.rb

```diff
@@ -27,7 +27,7 @@ class IntegrationTest < ActionDispatch::IntegrationTest
     assert_select("div", "bar")
     assert_response :success
 
-    baseline_response = response.body
+    baseline_response = response.body.gsub(/>[\s\n]+</, "><").strip
```

Comment:
These baseline controller renders included the whitespace and were failing the test because they weren't equal.